### PR TITLE
fix error "Can't find variable: DOMParser" in _connect_cb_wrapper

### PR DESCRIPTION
**Pull request by firemaples**
Native app shows "Can't find variable: DOMParser" in _connect_cb_wrapper in lib-jitsi-meet.min.js when connected to an Openfire server that we established.

Make DOMParser globally to resolve this problem

### CODE REVIEW

**Comment by saghul** [comment]
Can you please paste the full traceback?

**Comment by firemaples** [comment]
Can't find variable: DOMParser

_connect_cb_wrapper
lib-jitsi-meet.min.js:2:56473

dispatchEvent
event-target.js:172:43

<unknown>
WebSocket.js:237:27

emit
EventEmitter.js:182:12

__callFunction
MessageQueue.js:299:47

<unknown>
MessageQueue.js:111:26

__guard
MessageQueue.js:262:6

callFunctionReturnFlushedQueue
MessageQueue.js:110:17




device-2018-04-26-113642

File: react/features/base/lib-jitsi-meet/native/polyfills-browser.js [hunk 1]
```diff
@@ -99,6 +99,8 @@ function _visitNode(node, callback) {
 (global => {
     const { DOMParser } = require('xmldom');
 
+    global.DOMParser = DOMParser;
```
Comment (saghul) [inline react/features/base/lib-jitsi-meet/native/polyfills-browser.js]: Can you please add a comment and say that that lib-jitsi-meet requires it if using WebSockets? (see how we document other polyfills). Thanks!

Comment (firemaples) [reply react/features/base/lib-jitsi-meet/native/polyfills-browser.js]: I've added comment in 9b64424b29595361f3f7351325a01660ae519489

**Comment by firemaples** [comment]
I've added a comment.

**Comment by saghul** [comment]
Merged, cheers Louis!

**Comment by firemaples** [comment]
Cheers! 😁

**Pull request merged**